src/rpc/client.cpp

```diff
@@ -127,6 +127,7 @@ static const CRPCConvertParam vRPCConvertParams[] =
     { "gettxout", 1, "n" },
     { "gettxout", 2, "include_mempool" },
     { "gettxoutproof", 0, "txids" },
+    { "gettxoutsetinfo", 1, "hash_or_height" },
     { "lockunspent", 0, "unlock" },
```

Comment:
rpc_help.py looking for `{ "gettxoutsetinfo", 2, "no_index"}` here:

```
File "/bitcoin/test/functional/rpc_help.py", line 51, in run_test
    self.test_client_conversion_table()
File "/bitcoin/test/functional/rpc_help.py", line 69, in test_client_conversion_table
    raise AssertionError("RPC client conversion table ({}) and RPC server named arguments mismatch!\n{}".format(
AssertionError: RPC client conversion table (/bitcoin/src/rpc/client.cpp) and RPC server named arguments mismatch!
{('gettxoutsetinfo', 2, 'no_index')}
```

Reply:
Hm, strange I don't see this fail when running `rpc_help.py` locally. Should be fixed now.